pkg/driver/redis/device_manager_redis.go

```diff
@@ -699,6 +708,9 @@ func (d *DeviceManager) GetStorageKeys(u uuid.UUID) ([]byte, error) {
 	// hold our config
 	var b []byte
 	data, err := d.client.HGet(deviceStorageKeysHash, u.String()).Result()
+	if err != nil && errors.Is(err, redis.Nil) {
+		return nil, &common.NotFoundError{Err: fmt.Sprintf("device not found %s", u)}
```

Comment:
Maybe `storage keys for the device %s not found` will better describe the error here?

Reply:
Done